homework/06LessonSix.js

```diff
@@ -76,26 +80,29 @@ function ConferenceRoom() {
 // Add a method called sayHi() that returns "Hi, I'm " + this.name + '!';
 function Person(name) {
   this.name = name;
+  this.sayHi = function(){
+    return "Hi, I'm "  + this.name + '!';
+  }
 }
 
 // Problem 5:
 // create a new Person named Jen
-var jen;
+var jen = new Person("Jen");
 
 // Problem 6:
 // add jen to the conferenceRoom
 var conferenceRoom = new ConferenceRoom();
 // you can do this by calling the method enter() as follows:
 // conferenceRoom.enter(somePerson);
 // add your code for Problem 6 here
-
+conferenceRoom.enter(jen);
 
 // ****
 // Concept Checkpoint
 //
 // Write your answer in comments
 //
-// What is a method?
+// What is a method? A method is a piece of code that is called by name that is associated with an object.
```

Comment:
Close, but in this instance, a method is unique in that it's a working function inside a class.